src/query/test/compatibility/testdata/aggregators.test

```diff
@@ -240,24 +240,22 @@ load 5m
 	version{job="app-server", instance="0", group="canary"}		7
 	version{job="app-server", instance="1", group="canary"}		7
 
-# FAILING issue #14 (count_values)
-#eval instant at 5m count_values("version", version)
-#	{version="6"} 5
-#	{version="7"} 2
-#	{version="8"} 2
-
-
-#eval instant at 5m count_values(((("version"))), version)
-#       {version="6"} 5
-#       {version="7"} 2
-#       {version="8"} 2
-
-
-#eval instant at 5m count_values without (instance)("version", version)
-#	{job="api-server", group="production", version="6"} 3
-#	{job="api-server", group="canary", version="8"} 2
-#	{job="app-server", group="production", version="6"} 2
-#	{job="app-server", group="canary", version="7"} 2
+eval instant at 5m count_values("version", version)
+	{version="6"} 5
+	{version="7"} 2
+	{version="8"} 2
+
+eval instant at 5m count_values(((("version"))), version)
+       {version="6"} 5
+       {version="7"} 2
+       {version="8"} 2
+
+
+eval instant at 5m count_values without (instance)("version", version)
+	{job="api-server", group="production", version="6"} 3
+	{job="api-server", group="canary", version="8"} 2
+	{job="app-server", group="production", version="6"} 2
+	{job="app-server", group="canary", version="7"} 2
 
 # Overwrite label with output. Don't do this.
```

Comment:
Are these other ones still failing?

Reply:
Yes, they are more complicated to fix because we need to replace and merge values for duplicated tags/labels. I didn't want to increase complexity for this PR.